.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,23 @@
+name: PR build of Ballerina datamapper compiler extension
+
+on: pull_request
```

Comment:
Let's add push also since tokens are not shared with fork repo, it is best practise to send a PR form the upstream branch to master. Push event also needs to be added to see the build in the upstream branch before the merge.

Reply:
Added push event